src/store/index.ts

```diff
@@ -76,27 +76,36 @@ const {
     async fetchWorkspaces(context) {
       const { commit } = rootActionContext(context);
       const workspaces = await api.workspaces();
-      commit.setWorkspaces(workspaces);
+      commit.setWorkspaces(workspaces.results.map((h) => h.name));
     },
 
     async fetchWorkspace(context, workspace: string) {
       const { commit } = rootActionContext(context);
       commit.unsetCurrentWorkspace();
 
-      const nodeTables = await api.tables(workspace, { type: 'node' });
-      const edgeTables = await api.tables(workspace, { type: 'edge' });
       const graphs = await api.graphs(workspace);
+      const tables = (await api.tables(workspace)).results;
+      const nodeTables = tables.filter((table) => table.edge === false);
+      const edgeTables = tables.filter((table) => table.edge === true);
 
       commit.setCurrentWorkspace({
-        name: workspace, nodeTables, edgeTables, graphs,
+        name: workspace, nodeTables: nodeTables.map((h) => h.name), edgeTables: edgeTables.map((h) => h.name), graphs: graphs.results.map((h) => h.name),
```

Comment:
You'll want to put each key/value pair on its own line to improve readability here.

Reply:
See e4b6678